12-UsingCompositeCommands/UsingCompositeCommands.Core/UsingCompositeCommands.Core.csproj

```diff
@@ -13,7 +13,7 @@
     <DefaultLanguage>en-US</DefaultLanguage>
     <FileAlignment>512</FileAlignment>
     <ProjectTypeGuids>{786C830F-07A1-408B-BD7F-6EE04809D6DB};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
-    <TargetFrameworkProfile>Profile259</TargetFrameworkProfile>
+    <TargetFrameworkProfile>Profile49</TargetFrameworkProfile>
```

Comment:
This project needs to be updated from a PCL to something more modern... You'll want to delete the AssemblyInfo in this project... the csproj contents can be completely changed as follows... this will give you a multi-targeted Core project for the Composite Commands that could be reused outside of WPF if you wanted...

```xml
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFrameworks>netstandard2.0;net45</TargetFrameworks>
    <DefaultLanguage>en-US</DefaultLanguage>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Prism.Core" Version="7.2.0.1367" />
  </ItemGroup>

</Project>
```